webpack.config.js

```diff
@@ -1,16 +1,16 @@
 const path = require("path");
 
 module.exports = {
-  entry: path.join(__dirname, "example/app/Example.bs.js"),
+  entry: path.join(__dirname, "examples/1-basic/app/Basic.bs.js"),
```

Comment:
Eventually we'll want to have this `webpack` setup in each example dir and have them isolated, just like `urql`.